apps/src/studio/api.js

```diff
@@ -166,6 +166,14 @@ exports.changeScore = function (id, value) {
   Studio.queueCmd(id, 'changeScore', {'value': value});
 };
 
+exports.getScore = function () {
+  return Studio.playerScore;
+};
+
+exports.setScore = function (value) {
+  Studio.setScore(value);
```

Comment:
I need them to run immediately, because the commands from subsequent blocks evaluate their arguments as soon as they're enqueued, rather than when the queue is processed.

I tried using the queue, but then code like this wouldn't work:

`score = 50`
`move up by score`

It would move 0 pixels (or whatever the previous value of score was) instead of 50.